src/components/Checklist/Checklist.stories.tsx

```diff
@@ -49,18 +52,22 @@ const COLUMN_CHECKBOXES = Array.from({ length: 8 })
         tooltip: { content: `Random Tooltip ${index}` },
     }));
 
-export const Checklist: StoryFn<ChecklistProps> = (args: ChecklistProps) => {
+export const Checklist = (args: ChecklistProps) => {
     const [activeBoxes, setActiveBoxes] = useState<string[]>([]);
 
     return (
         <ChecklistComponent
             {...args}
+            ariaLabel={typeof args.ariaLabel === 'string' ? args.ariaLabel : 'checklist'}
```

Comment:
You can remove this, it's part of the args already.

Reply:
Removed